.ci/embedded-64bit-steps.yaml

```diff
@@ -0,0 +1,128 @@
+        wget --no-check-certificate \
+        "https://dl.bintray.com/boostorg/release/1.75.0/source/boost_1_75_0.tar.gz" && \
+        tar xvzf boost_1_75_0.tar.gz && \
+        rm -f boost_1_75_0.tar.gz && \
+
+    echo "Download and build OpenBLAS as a static library"
+    wget --no-check-certificate \
+      "https://github.com/xianyi/OpenBLAS/archive/v0.3.13.tar.gz" && \
+       tar xvzf v0.3.13.tar.gz && \
+       rm -f v0.3.13.tar.gz && \
+       cd OpenBLAS-0.3.13 && \
+       make TARGET=ARMV8 BINARY=64 HOSTCC=gcc CC=aarch64-linux-gnu-gcc FC=aarch64-linux-gnu-gfortran NO_SHARED=1 &&\
+       cd ../ &&\
+     
+    # Install armadillo.
+    curl https://data.kurg.org/armadillo-8.400.0.tar.xz | tar -xvJ
+
+    # Install cereal.
+    wget https://github.com/USCiLab/cereal/archive/v1.3.0.tar.gz
+    tar -xvzpf v1.3.0.tar.gz # Unpack into cereal-1.3.0/.
+    ls -la
+    pwd
+  displayName: 'Install Build Dependencies'
+
+# Configure mlpack (CMake)
+- script: |
+    unset BOOST_ROOT
+    mkdir build && cd build
+    if [ "$(binding)" == "go" ]; then
+      export GOPATH=$PWD/src/mlpack/bindings/go
+      go get -u -t gonum.org/v1/gonum/...
+    fi
+    cmake $(CMakeArgs) -DPYTHON_EXECUTABLE=`which python` ..
+  displayName: 'CMake'
+
+# Build mlpack
+- script: cd build && make
+  condition: eq(variables['CMakeArgs'], '-DDEBUG=ON -DPROFILE=OFF -DBUILD_PYTHON_BINDINGS=OFF -DBUILD_JULIA_BINDINGS=OFF -DBUILD_GO_BINDINGS=OFF -DBUILD_R_BINDINGS=OFF')
+  displayName: 'Build'
+
+# Build mlpack
+- script: cd build && make -j2
+  condition: ne(variables['CMakeArgs'], '-DDEBUG=ON -DPROFILE=OFF -DBUILD_PYTHON_BINDINGS=OFF -DBUILD_JULIA_BINDINGS=OFF -DBUILD_GO_BINDINGS=OFF -DBUILD_R_BINDINGS=OFF')
+  displayName: 'Build'
+
+- script: |
+    uname -m
+    docker pull arm64v8/ubuntu
+    docker run --rm -t arm64v8/ubuntu uname -m
+    docker pull multiarch/qemu-user-static
+    docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+  displayName: 'Download docker container'
+ 
+# #Allow running docker container with different architecture than the host
+# - task: Docker@2
+#   displayName: 'Run privileged container'
+#   inputs:
+#     command: run
+#     arguments: '--rm --privileged multiarch/qemu-user-static:register --reset -p yes'
+#   enabled: false
+
+# Start docker container
+- task: Docker@2
+  displayName: 'Run tests via ctest'
+  inputs:
+    command: run
+    arguments: '-v /home/vsts/work/1/s:/opt/share -t arm64v8/ubuntu'
+    tags: arm64v8/ubuntu
+    script: | # maybe the name is different?
+        uname -m
+        ls
+        whoami
+  #addPipelineData: false
+
+# Run tests via ctest.
+#- script: |
+    #docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+    #docker run -v $PWD:/opt/share -t arm64v8/ubuntu
+#    apt-get update
+#    apt-get install -y --allow-unauthenticated cmake
+#    cd /opt/share/build && CTEST_OUTPUT_ON_FAILURE=1 ctest -T Test .
+#  displayName: 'Run tests via ctest'
```

Comment:
@zoq After the tags, it entered this loop, no information is printed for line 100. Any ideas?